x/ibc/testing/chain.go

```diff
@@ -634,7 +662,7 @@ func (chain *TestChain) ConnectionOpenTry(
 	proofConsensus, consensusHeight := counterparty.QueryConsensusStateProof(counterpartyConnection.ClientID)
 
 	msg := connectiontypes.NewMsgConnectionOpenTry(
-		connection.ID, connection.ID, connection.ClientID, // testing doesn't use flexible selection
+		connection.ID, connection.ClientID, // testing doesn't use flexible selection
```

Comment:
```suggestion
		connection.ID, connection.ClientID,
```

Isn't the choice between flexible and fixed selection removed in this pr?